### ISSUE TITLE
[EN][Quest][Dun Morogh] Make Hay While the Sun Shines

### ISSUE THREAD

**Issue by nt32** [Atlantiss/BugTracker#2054]
Hello, there are some issues with chain quest in general but following quest at dwarven starting zone http://www.wowhead.com/quest=24486/make-hay-while-the-sun-shines
is not offered to players at any point which makes it impossible to continue with questing at that zone.
Can you please fix that please ? it makes it hard for new players to level properly.

**Comment by LorianoCZ** [comment]
Yeah, this is pretty annoying and I can not level my dwarf past lvl 3. Please fix it.
